### PR TITLE
Updates

### PR DESCRIPTION
**Pull request by Link4K**

### CODE REVIEW

File: channels/sd.m3u [hunk 1]
```diff
@@ -9,3 +9,9 @@ http://africatv.live.net.sa:1935/live/africatv3/playlist.m3u
 http://africatv.live.net.sa:1935/live/africatv3/playlist.m3u8
 #EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Arabic" tvg-logo="http://mahgoubsons.com/wp/wp-content/uploads/2017/10/BNTV_logo.png" group-title="",Blue Nile
 http://livecdnh2.tvanywhere.ae/hls/blue_nile/04.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Arabic" tvg-logo="" group-title="",Sudan TV
+http://217.182.137.206/sudantv.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Arabic" tvg-logo="" group-title="",Sudania 24 TV
+http://217.182.137.206/sudania24.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Arabic" tvg-logo="" group-title="",Sudan Drama
+http://217.182.137.206/sudan_drama.m3u8
```
Comment (freearhey) [inline channels/sd.m3u]: These links don't work anymore.

**Review by freearhey — CHANGES_REQUESTED** [review]
Please remove broken links from PR so I can approve it.

File: channels/sd.m3u (hunk 1 above)
Comment (Link4K) [reply channels/sd.m3u]: Work in Microsoft Edge & streamlink

> C:\Users>streamlink hlsvariant://http://217.182.137.206/sudan_drama.m3u8 best
[cli][info] Found matching plugin hls for URL hlsvariant://http://217.182.137.206/sudan_drama.m3u8
[cli][info] Available streams: live (worst, best)
[cli][info] Opening stream: live (hls)
[cli][info] Starting player: "C:\Program Files\VideoLAN\VLC\vlc.exe"
[cli][info] Player closed
[cli][info] Stream ended
[cli][info] Closing currently open stream...
C:\Users>

Comment (freearhey) [reply channels/sd.m3u]: I just double-checked the links using VLC Player and streamtest.in.
Captura de pantalla 2020-03-01 a las 13 02 58
Captura de pantalla 2020-03-01 a las 13 04 02

Comment (Link4K) [reply channels/sd.m3u]: ok removed.

**Pull request merged**